### ISSUE TITLE
Demo site is not working anymore

### ISSUE THREAD

**Issue by xavierfoucrier** [andreupifarre/preload#1]
It seems that the **Github pages site is not working anymore** since 1.2.2 release: https://unpkg.com/preload-it@1.2.2/index.js returns 404.

**Comment by andreupifarre** [comment]
@xavierfoucrier The index.js is not intended to be in the _dist_ folder. I have double checked the demo site and I cannot see any 404 errors, can you please provide the URL that you mentioned?
You can check the folder structure here: https://unpkg.com/preload-it@latest/

**Comment by xavierfoucrier** [comment]
Hi @andreupifarre!

The problem is that the demo is not properly working on **Github pages** at https://andreupifarre.github.io/preload-it/docs/index.html, here is a screenshot of the 404:

![capture](https://user-images.githubusercontent.com/2471223/50293337-a48d8900-0473-11e9-8e8b-7b4d53cfd43a.JPG)

:wink:

**Comment by andreupifarre** [comment]
@xavierfoucrier Interestingly I don't get that 404 error on https://andreupifarre.github.io/preload-it/docs/index.html, not sure what is going on.

![screenshot 2018-12-20 at 16 31 18](https://user-images.githubusercontent.com/6190486/50297470-c8050380-0474-11e9-9266-5f56c32e4d17.png)

**Comment by xavierfoucrier** [comment]
Hum... weird.
I am investigating further.

**Comment by xavierfoucrier** [comment]
@andreupifarre can you try to reload the site without using the browser cache? May be its a problem with the unpkg CDN, but calling https://unpkg.com/preload-it always returns a 404. I have try to run the site on different browsers too.

**Comment by andreupifarre** [comment]
@xavierfoucrier It all seems to work fine, I tried downloading https://unpkg.com/preload-it from several clients including chrome with emptied cache, terminal `curl` and postman, they all returned the correct .js file.

**Comment by xavierfoucrier** [comment]
The problem is that the demo page call the **preload-it package** like this:

```html
<script src="https://unpkg.com/preload-it"></script>
```

This URL from the **unpkg CDN** redirect from `https://unpkg.com/preload-it` to the latest package hosted at `https://unpkg.com/preload-it@1.2.2`, then the CDN finally redirect to `https://unpkg.com/preload-it@1.2.2/index.js` which doesn't exist and output `Cannot find "/index.js" in preload-it@1.2.2`.

May be I am wrong, but the script src should be writed like this:

```html
<script src="https://unpkg.com/preload-it@latest/dist/preload-it.min.js"></script>
```

:coffee:

**Comment by andreupifarre** [comment]
@xavierfoucrier if you open https://unpkg.com/preload-it in your browser, what do you see?
It should redirect to the latest version, whichever it is, currently it is https://unpkg.com/preload-it@1.2.2/dist/preload-it.min.js
I don't get why you are being redirected to https://unpkg.com/preload-it@1.2.2/index.js instead.
Do you have any extension installed on your browser? Could you try with another browser?

**Comment by xavierfoucrier** [comment]
@andreupifarre if I open the link you provide I exactly get: Cannot find "/index.js" in preload-it@1.2.2.

I think the problem comes from **unpkg CDN** where your main entry point is define as `index` instead of `preload-it`... I have check your package.json and it's fine, I can't find any reference of `index.js` in your source and finally I don't understand too. :confused:

![capture](https://user-images.githubusercontent.com/2471223/50355528-4a61f600-054f-11e9-8ab8-3f2c9a378a01.PNG)

I have tried with different browsers... always the same result.

Note that I have tried to get the latest version of a package I have coded and it works just fine: calling `https://unpkg.com/instacam` return `https://unpkg.com/instacam@1.6.2` then `https://unpkg.com/instacam@1.6.2/src/instacam.js` which is the main entry point of the library.
